helm/frconfig/values.yaml

```diff
@@ -4,6 +4,13 @@
 
 replicaCount: 1
 
+
+config:
+  name: frconfig
```

Comment:
So the way I read this is that users will still deploy the frconfig chart and it's basically a container for the Git secret and the details of the Git repository. 
Also, when redeploying AM/IDM/IG in the event of a config change, seems like there's no longer the need to reinstall the frconfig chart.
Could you confirm my understanding is correct? Thanks.

Reply:
Correct. 
It also means that if they want different config repos (for say IG and IDM), they can deploy another instance of frconfig, with a different config name.